src/content/advanced/configure-dex-in-your-cluster/index.md

```diff
@@ -193,9 +176,47 @@ data:
             redirectURI: https://dex.<CLUSTERID>.<BASEDOMAIN>/callback
 ```
 
+{{< /tab >}}
+{{< tab title="Okta">}}
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: dex-app-user-values
+  namespace: <CLUSTERID>
+data:
+  values: |
+    isWorkloadCluster: true
+    oidc:
+      expiry:
+        signingKeys: 6h
+        idTokens: 30m
+      customer:
+        enabled: true
+        connectors:
+        - id: customer
+          connectorName: test
+          connectorType: oidc
+          connectorConfig: >-
+            clientID: <CLIENT-ID-SET-IN-YOUR-IdP>
+            clientSecret: <CLIENT-SECRET-SET-IN--YOUR-IdP>
+            insecureEnableGroups: true
+            insecureSkipEmailVerified: true
```

Comment:
Sorry @pipo02mix I didn't realise this is a generic guide. In that case `insecureSkipEmailVerified` should go to the warning section. I think same applies to `insecureEnableGroups`. I'm not even sure if we needed to add this in our last case.